dvc/fs/fsspec_wrapper.py

```diff
@@ -263,22 +255,18 @@ class CallbackMixin:
     """Use the native ``get_file()``/``put_file()`` APIs
     if the target filesystem supports callbacks."""
 
-    def _upload(
-        self, from_file, to_info, name=None, no_progress_bar=False, **pbar_args
+    @staticmethod
+    def put_file_compat(
```

Comment:
Temporary, I'm planning to get rid of `_upload`/`_download` from `CallbackMixin` and then this `CallbackMixin` will just be a way to provide compat support for callback. Naming is bad, but will just be replace by `put_file` after I get rid of `_download`.